src/dbnode/storage/repair.go

```diff
@@ -714,7 +714,12 @@ func (r shardRepairer) shadowCompare(
 		peerM = dynamic.NewMessage(nsCtx.Schema.Get().MessageDescriptor)
 	}
 
-	contextPool := r.Options().AdminClient().Options().ContextPool()
+	// TODO(bodu): What do we do when there's more than one admin client configured?
+	adminClients := r.Options().AdminClients()
```

Comment:
@robskillington this allows for setting of multiple admin clients, not sure what the right thing to do here is.

Reply:
Will f/u on this once merged and rebased on master.